src/components/dialogs/parameters/parameters.jsx

```diff
@@ -130,7 +130,7 @@ export const styles = {
     singleItem: (theme) => ({
         display: 'flex',
         flex: 'auto',
-        alignItems: 'center',
+        alignItems: 'flex-start',
```

Comment:
why ?

Reply:
center the elements of the container may cause form validation error messages to appear misaligned.
the form is looking better with  'flex-start':

![image](https://github.com/user-attachments/assets/66162477-9d36-4723-a512-950c676f45e3)

![image](https://github.com/user-attachments/assets/21a7438e-6137-4fab-9523-3ec9ce48a400)